101-sqlmi-new-vnet/README.md

```diff
@@ -0,0 +1,14 @@
+# Azure Sql Database Managed Instance (SQL MI) Creation inside New Virtual Network
```

Comment:
Why are we skipping CI?

Reply:
SQL MI is still in gated public preview so the subscription needs to be whitelisted for deployment. Deployment takes more than 1-2 days.

Reply:
Whitelist the CI subscription and manually validate the deployment

Reply:
Validated deployment manually.

Reply:
ok. lmk when the sub is whitelisted and I will test manually